manager/network/cni/cni.go

```diff
@@ -66,8 +51,7 @@ func (nm *cni) ValidateNetworkSpec(spec *api.NetworkSpec) error {
 	}
 
 	if spec.Annotations.Name != cni.Network.Name {
-		return grpc.Errorf(codes.InvalidArgument,
-			"CNI Network name (%q) must match Spec annotations name (%q)",
+		log.G(ctx).Infof("CNI Network name (%q) does not match Spec name (%q)",
```

Comment:
Seems okay to log the mismatch, though I'm not sure users will really notice this in the manager log.